### PR TITLE
feat(VantMarkdownVetur): support arguments of events

### PR DESCRIPTION
**Pull request by nemo-shen**
#10422

- [x] transform '，' to ',' in README.md
- [x] support `events.arguments` compile to `web-types.json`
- [x] adjust part of `events.arguments` content in `README.md` & `README.zh-CN.md`
- [x] fixed multi '\|' can not transform to 'JOIN' in packages/vant-markdown-vetur/src/parser.ts

### Additions:
I discover so many regexp used in `vant-markdown-vetur` when support the feature that is understand difficulty. so I wonder if we should reduce the use of such. other, `vant-markdown-vetur` no unit test, we need to support it to improve the reliability.

### CODE REVIEW

**Comment by nemo-shen** [comment]
Wow! unbelievable! too many cases need to match...😲

File: packages/vant/src/address-list/README.md [hunk 1]
```diff
@@ -93,14 +93,7 @@ export default {
...
 
-| Event | Description | Arguments |
-| --- | --- | --- |
-| add | Emitted when the add button is clicked | - |
-| edit | Emitted when the edit icon of address is clicked | _item: Address, index: number_ |
-| select | Emitted when an address is selected | _item: Address, index: number_ |
-| edit-disabled | Emitted when the edit icon of disabled address is clicked | _item: Address, index: number_ |
-| select-disabled | Emitted when a disabled address is selected | _item: Address, index: number_ |
-| click-item | Emitted when an address item is clicked | _item: Address, index: number_ |
+| Event | Description | Arguments | | --- | ``--- | --- | | add | Emitted when the add button is clicked | - | | edit | Emitted when the edit icon of address is clicked | _item: Address, index: number_ | | select | Emitted when an address is selected | _item: Address, index: number_ | | edit-disabled | Emitted when the edit icon of disabled address is clicked | _item: Address, index: number_ | | select-disabled | Emitted when a disabled address is selected | _item: Address, index: number_ | | click-item | Emitted when an address item is clicked | _item: Address, index: number_ |
```
Comment (chenjiahan) [inline packages/vant/src/address-list/README.md:96]: 这里格式炸了

Comment (nemo-shen) [reply packages/vant/src/address-list/README.md:96]: 对的，格式炸裂了，我在兼容格式呢~

Comment (nemo-shen) [reply packages/vant/src/address-list/README.md:96]: argmentus里以前有好多中写法

Comment (chenjiahan) [reply packages/vant/src/address-list/README.md:96]: 多了一个 ``

File: packages/vant/src/collapse/README.md [hunk 2]
```diff
@@ -114,23 +114,23 @@ export default {
...
 | border | Whether to show outer border | _boolean_ | `true` |
 
 ### Collapse Events
 
-| Event  | Description                  | Arguments   |
-| ------ | ---------------------------- | ----------- |
-| change | Emitted when switching panel | activeNames |
+| Event  | Description                  | Arguments           |
+| ------ | ---------------------------- | ------------------- | ------ | ------------ | ------- |
+| change | Emitted when switching panel | activeNames: string | number | Array<string | number> |
```
Comment (chenjiahan) [inline packages/vant/src/collapse/README.md:125]: 这里格式化结果异常了

`activeNames: string | number | Array<string | number>`

要写成

`_activeNames: string \| number \| Array<string \| number>_`

**Comment by chenjiahan** [comment]
CI 挂咯

**Comment by nemo-shen** [comment]
😂等下到家看 电脑关机了 真尴尬

**Review by chenjiahan — APPROVED** [review]

**Comment by chenjiahan** [comment]
👏🏻👏🏻

**Pull request merged**